pyvista/examples/vrml.py

```diff
@@ -0,0 +1,58 @@
+"""VRML examples."""
+
+from .downloads import _retrieve_file
+
+VRML_SAMPLES_ROOT_URL = "https://raw.githubusercontent.com/lorensen/VTKExamples/master"
```

Comment:
Agree with isolating the examples into a separate module. Our examples module is getting too large.